packages/ui/pages/index.tsx

```diff
@@ -53,19 +53,21 @@ export const TASK_SUB_TABLE_HEADINGS = [
     title: 'End Date',
     dataIndex: 'end_date',
     key: 'end_dates',
-  }
+  },
 ];
 const USER_NAME = 'Ramya';
 const Home = () => {
-  const router = useRouter()
-  const programsContext = useProgramsContext()
-  const isFirstTimeVisitor = !programsContext.programs.length
-  
-  if (isFirstTimeVisitor) {
-    router.push('/workspace/stepform')
-    return null
-  }
-  
+  const router = useRouter();
+  const programsContext = useProgramsContext();
+  const isFirstTimeVisitor = !programsContext.programs.length;
+
+  useEffect(() => {
+    if (isFirstTimeVisitor) {
+      router.push('/workspace/stepform');
+      return null;
```

Comment:
@venkatram090993 I think redirecting to `/workspace/stepform` is enough no need to return `null`. 
The `useEffect` hook returns only functions which will be used later for cleanup when the component is Unmounting.